galpy/potential/amuse.py

```diff
@@ -1,10 +1,13 @@
 # galpy.potential.amuse: AMUSE representation of galpy potentials
 import numpy
-from amuse.units import units
+from amuse.units import units as u
 from amuse.units.quantities import ScalarQuantity
 from amuse.support.literature import LiteratureReferencesMixIn
 from .. import potential
 from ..util import bovy_conversion
+from .Force import _APY_LOADED
+if _APY_LOADED:
+    from astropy import units as apy_u
```

Comment:
I think this now all looks good, but I don't think ``astropy.units`` is necessary anymore, so could you please switch back to the old form (where ``amuse.units`` is imported as ``units``)?

Also, the AMUSE tests aren't actually run on travis, because it's too difficult to install AMUSE on travis, so we just run these locally from time to time to make sure that they pass.

Reply:
done